apps/journeys-admin/src/components/TemplateView/TemplateViewHeader/TemplateViewHeader.spec.tsx

```diff
@@ -70,7 +55,7 @@ describe('TemplateViewHeader', () => {
     expect(getByRole('img', { name: 'image.jpg' })).toBeInTheDocument()
   })
 
-  it('should display the featured date, title, and description', () => {
+  it('should display the  title, and description', () => {
```

Comment:
theres a double space in here 
```suggestion
  it('should display the title and description', () => {
```